enhanced-resolve/enhanced-resolve-tests.ts

```diff
@@ -0,0 +1,42 @@
+import resolve = require('enhanced-resolve');
+import { Context } from './lib/concord';
```

Comment:
Prefer to import `enhanced-resolve/lib/concord` because that's how `@types` users will have to write it.